integrations/access/msteams/configure.go

```diff
@@ -45,19 +45,23 @@ var (
 	zipFiles = []string{"manifest.json", "outline.png", "color.png"}
 )
 
-// payload represents template payload
-type payload struct {
+// Payload represents template Payload used by the manifest and config templates.
+type Payload struct {
```

Comment:
As this struct bercomnes public, its name should describe what it is.

`msteams.Payload` makes little sense for people who don't know the Teams plugin